### ISSUE TITLE
change visulaization options

### ISSUE THREAD

**Issue by bistek** [pyvista/pyvista-support#218]
Hello,

this is probably a stupid question, but I cannot find a solution. Is it possible to modify the color, colormap, property shown as colormap, transparency, etc. of an actor in a view AFTER having shown it?

If I check the attributes of an actor (obtained showing a mesh in a plotter), I cannot find something like "color" (even if there are methods like SetTexture and GetTexture).

In other words, I can do:

`my_actor = my_plotter.add_mesh(my_mesh, show_edges=True, color=[0, .8, .8])`

but I do not find a way to do

`my_actor.color = [0, .8, .8]`

after the actor is shown.

Thanks very much!

**Comment by GuillaumeFavelier** [comment]
What an interesting question! I would also be very interested to know the best way to achieve this with PyVista :)

Let me share my hacks:

1) First, I would add the mesh and give it a `name` with:

```py
my_plotter.add_mesh(my_mesh, show_edges=True, color=[0, .8, .8], name="MyMesh")
```

Then, with the `new_color`:

```py
my_plotter.add_mesh(my_mesh, show_edges=True, color=new_color, name="MyMesh")
```

But this solution will probably replace the actor in the scene.

2) I would use the actor properties:

```py
my_actor = my_plotter.add_mesh(my_mesh, show_edges=True, color=[0, .8, .8])
my_actor.GetProperty().SetColor(new_color)
```

But sadly, this is far from being an elegant general solution.

Honestly something as easy as:

```py
my_actor.color = [0, .8, .8]
```

would be awesome!

**Comment by bistek** [comment]
Ok, so at least my_actor.GetProperty() returns the properties, but which properties can be accessed like this?

Thanks!

**Comment by GuillaumeFavelier** [comment]
Those come from [`vtkProperty`](https://vtk.org/doc/nightly/html/classvtkProperty.html) and AFAIK this class is not wrapped by PyVista so I recommend to use with caution :)

**Comment by banesullivan** [comment]
> AFAIK this class is not wrapped by PyVista so I recommend to use with caution 

Yep, at that point you are using raw VTK so you'll have to refer to the VTK documentation. You are more than welcome to request (in an issue in PyVista) that we wrap the `vtkActor` and `vtkProperty` class to make accessing/changing those easier but I'm not sure if any of the core devs have the bandwidth or need to wrap those classes. Though it'd be good to document the need and use case if someone out there also has the need and are willing to implement it

**Comment by T4mmi** [comment]
I'm also in the same case, though I'd like to be able to choose between a solid color `SetColor()` and a scalar array ...
still doing it the `add_mesh(...)` way but this is suboptimal since only the color is changing but it recreate the whole actor ...

**Comment by renefritze** [comment]
Replacing the mesh for me also to reset the position of an interactive scalar bar. So I too would prefer to only be able to change the mesh-associated scalar array.
